### DISCUSSION TITLE
Error login

### DISCUSSION THREAD

**Discussion by thanhnguyen2075** [dishapatel010/gdriveDL#3] [General]
Excuse me!
i met error
500. That’s an error.
There was an error. Please try again later. That’s all we know.
how to fix.
Thanks you!

**Comment by dishapatel010** [comment]
> Excuse me! i met error 500. That’s an error. There was an error. Please try again later. That’s all we know. how to fix. Thanks you!

Can you tell me more on what are you trying, else read README.md everything is solved.
